apps/src/lib/node/ledger/ethereum_oracle/events.rs

```diff
@@ -96,34 +89,15 @@ pub mod eth_events {
                         nonce: nonce.parse_uint256()?,
                         transfers: transfers
                             .parse_transfer_to_namada_array()?,
-                        valid_transfers_map: valid_map,
                     }
                 }
-                RawEvents::Governance(GovernanceEvents::NewContractFilter(
```

Comment:
So how are we planning to upgrade contracts without these events?

Reply:
new contracts/contract upgrades were not being handled anyway. I guess the answer to this is a hard fork, if it ever comes to it

Reply:
we are going to make a governance proposal on namada (with wasm code to to change the storage)